src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace WCFClientStressTests
+{
+    public class Program
+    {
+        public void Main(string[] args)
+        {
+            ProcessRunOptions(args);
+
+            DoTheRun(3000000);
+
+            Console.WriteLine("Done. Press Enter to GC.");
+            Console.ReadLine();
+            GC.Collect();
+            Console.WriteLine("After GC");
+            Console.ReadLine();
+        }
+
+        private void ProcessRunOptions(string[] args)
+        {
+            // All hardcoded right now, but we need to extract the following from the parameters:
+            //  -   HostName
+            //  -   AppName
+            //  -   TCP/HTTP
+            //  -   duration and/or # of iterations to run
+            //  -   # factories to pool
+            //  -   # channels to pool
+            //  -   etc
```

Comment:
Optional -- open an issue to make the changes described here.  We try to avoid any kind of "TODO" statements in OSS code.